templates/components/products/price.html

```diff
@@ -1,37 +1,83 @@
-{{#if price.with_tax}}
-    <div class="price-section price-section--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
-        {{#if price.rrp_with_tax}}
-            <span data-product-rrp-with-tax class="price price--rrp">{{price.rrp_with_tax.formatted}}</span>
-        {{/if}}
-        <span data-product-price-with-tax class="price">{{price.with_tax.formatted}}</span>
-        {{#if price.without_tax}}
-            {{lang 'products.price_with_tax' tax_label=price.tax_label}}
-        {{/if}}
-    </div>
-{{/if}}
-
-{{#if price.without_tax}}
-    <div class="price-section price-section--withoutTax {{#if price.with_tax}}price-section--minor{{/if}}"  {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
-        {{#if price.rrp_without_tax}}
-            <span data-product-rrp-without-tax class="price price--rrp">{{price.rrp_without_tax.formatted}}</span>
-        {{/if}}
-        {{#if schema_org}}
-            <meta itemprop="price" content="{{price.without_tax.value}}">
-            <meta itemprop="priceCurrency" content="{{currency_selector.active_currency_code}}">
-        {{/if}}
-        <span data-product-price-without-tax class="price price--withoutTax">{{price.without_tax.formatted}}</span>
-        {{#if price.with_tax}}
-            <abbr title="{{lang 'products.excluding_tax'}}">{{lang 'products.price_without_tax' tax_label=price.tax_label}}</abbr>
+{{#or has_options product.options.length}}
+    {{#if price.with_tax}}
+        <div class="price-section price-section--withTax" {{#if schema_org}}itemprop="offers" itemscope itemtype="http://schema.org/Offer"{{/if}}>
+            {{#if price.rrp_with_tax}}
+                <span data-product-rrp-with-tax class="price price--rrp">{{price.rrp_with_tax.formatted}}</span>
+            {{/if}}
+            {{#if theme_settings.price_as_low_as}}
+                <span translate>As low as </span>
```

Comment:
pulled this out from within the span (and below too) because updatePriceView in product-details.js will overwrite everything inside the span